packages/@webex/plugin-cc/test/unit/spec/services/task/TaskManager.ts

```diff
@@ -10,15 +10,6 @@ import { TASK_EVENTS } from '../../../../../src/services/task/types';
 import WebCallingService from '../../../../../src/services/WebCallingService';
 import config from '../../../../../src/config';
 
-jest.mock('./../../../../../src/services/task', () => {
```

Comment:
question. Why have we removed this mock?

Reply:
This is not needed anymore